core/storage/question/gae_models.py

```diff
@@ -768,6 +768,7 @@ class QuestionSummaryModel(base_models.BaseModel):
     # ids in the question.
     misconception_ids = (
         datastore_services.StringProperty(indexed=True, repeated=True))
+    version = datastore_services.IntegerProperty(required=True)
```

Comment:
Do we index version properties in other summary models? (Can you please check whether we do it in other summary models like ExpSummaryModel?)

Reply:
No we don't.

https://github.com/oppia/oppia/blob/58c610d01bd72e313634a38d90c037ade64b3ad2/core/storage/exploration/gae_models.py#L1082